addons/im_livechat/static/src/embed/common/chat_window_patch.js

```diff
@@ -18,9 +18,14 @@ patch(ChatWindow.prototype, {
         this.livechatState = useState({ showCloseConfirmation: false });
     },
 
-    async close() {
+    async close({ withConfirmation = true } = {}) {
         const chatWindow = toRaw(this.props.chatWindow);
-        if (chatWindow.thread.id > 0 && !this.livechatState.showCloseConfirmation) {
+        if (
+            withConfirmation &&
+            !chatWindow.hasFeedbackPanel &&
+            chatWindow.thread.id > 0 &&
+            !this.livechatState.showCloseConfirmation
+        ) {
```

Comment:
Default should be `withConfirmation = false`, because that's the default with the rest of `mail` code in backend or public page (non-livechat visitor PoV).

Relying on different behaviour in bundle, while this is quick and easy for now, adds significant difficulties to combine features in all contexts. Thus it's best to try to design current and new features to ease combining them as this is very likely desireable to share most features in all contexts.